README.md

```diff
@@ -37,26 +37,24 @@ Build (install [nix](#about-the-development-environment) first)
 
 ```sh
 nix develop
-nix build '.?submodules=1'
+nix build
```

Comment:
did you fix this so `'.?submodules=1'` is no longer necessary?

Reply:
I could've sworn you said this was all that was required... Never mind, will revert